types/wordpress__components/button/index.d.ts

```diff
@@ -44,8 +47,42 @@ declare namespace Button {
          * Renders a button with an anchor style.
          */
         isLink?: boolean;
+        /**
+         * Renders a disabled button.
+         */
         disabled?: boolean;
         className?: string;
+        /**
+         * The icon to render. Supported values are: Dashicons (specified as
+         * strings), functions, WPComponent instances and `null`.
+         * @defaultValue null
+         */
+        icon?: Icon.Props<any>['icon'];
+        /**
+         * The size (width and height) of the icon.
+         * @defaultValue `20` (when using Dashicon), `24` otherwise
+         */
+        iconSize?: Icon.Props<any>['size'];
+        /**
+         * Renders a with the `label` prop when `true`.
+         * @defaultValue false
+         */
+        showTooltip?: boolean;
+        /**
+         * The direction in which the tooltip should open relative to its
+         * parent node.
+         */
+        tooltipPosition?: Tooltip.Props['position'];
+        shortcut?: Tooltip.Props['shortcut'];
+        /**
+         * Used as the Tooltip text and `aria-label` if one is not provided.
+         */
+        label?: string;
+        /**
+         * When `true` will allow a disabled button to still be perceivable
+         * by screen reader users.
+         */
+        __experimentalIsFocusable?: boolean;
```

Comment:
😬  Should experimental APIs be included? These tend to be volatile. It might be better to only document the stable API.

Reply:
I explicitly did not include experimental APIs when I wrote the initial types for this reason. I'll leave to you all though to make the call as to whether or not to add them in.

Reply:
I wasn't sure, so erred on the side of including it. I'll remove it though, agreed it seems risky to document it in this way!